src/components/TreeNode.tsx

```diff
@@ -231,7 +303,31 @@ const TreeNode = ({
           top: y,
           left: 0,
         }
-      : undefined
+      : isMoveAnimating
+        ? moveType === 'moveThoughtCursor'
+          ? {
+              transformOrigin: 'left',
+              transform: 'scale3d(1.5, 1.5, 1)',
+              transition: `transform {durations.layoutNodeAnimation} ease-out`,
+            }
+          : moveType === 'moveThoughtSibling'
+            ? {
+                transformOrigin: 'left',
+                transform: 'scale3d(0.5, 0.5, 1)',
+                filter: 'blur(2px)',
+                opacity: 0,
```

Comment:
Let's not fade to `opacity: 0`, as that makes it look like the thought has completely disappeared. They have swapped places and should both always be visible.

Reply:
This is now using `opacity: 0.5` so that it stays visible.